app/jobs/people/probe_duplicate_job.rb

```diff
@@ -0,0 +1,10 @@
+# encoding: utf-8
+
+class People::ProbeDuplicateJob < RecurringJob
```

Comment:
same name as finder class ...

People::DuplicateFinderJob

no need to create an additional name here